app/controllers/animals_controller.rb

```diff
@@ -0,0 +1,30 @@
+class AnimalsController < ApplicationController
+  def index
+    render 'static_pages/index'
+  end
+
+  def show
+    render 'static_pages/index'
+  end
+
```

Comment:
this is totally fine, if you wanted to you could also in routes.rb have requests with these paths go immediately to the `static_pages_controller#index`. You will likely be doing that often anyways. Up to you, just try to be consistent